linode/manager
# upcoming: [DI-20929] - Added applied filters view in CloudPulse

## Description 📝

Added a applied filters view in CloudPulse

## Changes  🔄

List any change(s) relevant to the reviewer.

1. Added new components CloudPulseAppliedFilterRenderer & CloudPulseAppliedFilter
2. Updated filter change handler to accept label parameter that will allow to pass labels that should be shown on applied filters
3. Added test cases for CloudPulseAppliedFilter component & also updated test cases that were failing due to parameter changes of handler.

Note: E2E tests for applied filter will be raised as separate PR, once this will be merge.

## Target release date 🗓️

12th Dec

## Preview 📷

**Include a screenshot or screen recording of the change.**


https://github.com/user-attachments/assets/37947f11-a4fd-4f76-b6c4-b20dd6dc48bc

Mobile View
![Screenshot 2024-12-03 at 7 18 12 PM](https://github.com/user-attachments/assets/38485ee3-b0d8-4716-af21-5aa6e1c49c3c)


## How to test 🧪

1. Switch to mock user
2. Select values from global filters
3. collapse the global filters by clicking filter button
4. You'll see selected filters in the same order as they are in the drop down. Also it'll be shown in 2 rows and after that it'll be scrollable

<details>
<summary> Author Checklists </summary>

## As an Author, to speed up the review process, I considered 🤔

👀 Doing a self review
❔ Our [contribution guidelines](https://github.com/linode/manager/blob/develop/docs/CONTRIBUTING.md)
🤏 Splitting feature into small PRs
➕ Adding a [changeset](https://github.com/linode/manager/blob/develop/docs/CONTRIBUTING.md#writing-a-changeset)
🧪 Providing/improving test coverage
 🔐 Removing all sensitive information from the code and PR description
🚩 Using a feature flag to protect the release
👣 Providing comprehensive reproduction steps
📑 Providing or updating our documentation
🕛 Scheduling a pair reviewing session
📱 Providing mobile support
♿  Providing accessibility support

<br/>


## As an Author, before moving this PR from Draft to Open, I confirmed ✅


</details>
